hikari/api/rest.py

```diff
@@ -242,7 +243,9 @@ async def edit_channel(
         name : hikari.undefined.UndefinedOr[str]
             If provided, the new name for the channel.
         flags : hikari.undefined.UndefinedOr[hikari.channels.ChannelFlag]
-            If provided, the new flags for the channel
+            If provided, the new channel flags to use for the channel. This can
+            only be used on a forum channel to apply ChannelFlag.REQUIRE_TAG, or
+            on a forum thread to apply ChannelFlag.PINNED.
```

Comment:
```suggestion
            If provided, the new channel flags to use for the channel. This can
            only be used on a forum channel to apply `REQUIRE_TAG`, or
            on a forum thread to apply `PINNED`.
```